ReactiveCocoa/Swift/SignalProducer+Restart.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  SignalProducer+Restart.swift
```

Comment:
@filwag agreed, updated in commit https://github.com/smilypeda/ReactiveCocoa/pull/1/commits/25c745b925247bf22d7af274b1989c374ca79f28